authz/providers/azure/rbac/checkaccessreqhelper_test.go

```diff
@@ -1046,39 +1151,63 @@ func Test_buildCheckAccessURL(t *testing.T) {
+			want:          "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789abc/resourceGroups/testResourceGroup/providers/Microsoft.Provider/resourceTypes/resourceName/providers/Microsoft.Authorization/checkaccess?api-version=2018-09-01-preview",
 		},
 		{
 			name:          "valid with namespace",
 			baseURL:       mustCreateURL("https://management.azure.com"),
 			resourceID:    testAzureResourceID,
 			hasNamespace:  true,
 			namespacePath: namespaces + "/dev",
-			want:          "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789012/resourceGroups/testResourceGroup/providers/Microsoft.Provider/resourceTypes/resourceName/namespaces/dev/providers/Microsoft.Authorization/checkaccess?api-version=2018-09-01-preview",
+			want:          "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789abc/resourceGroups/testResourceGroup/providers/Microsoft.Provider/resourceTypes/resourceName/namespaces/dev/providers/Microsoft.Authorization/checkaccess?api-version=2018-09-01-preview",
 		},
 		{
 			name:          "valid with managed namespace",
 			baseURL:       mustCreateURL("https://management.azure.com"),
 			resourceID:    testAzureResourceID,
 			hasNamespace:  true,
 			namespacePath: managedNamespaces + "/dev",
-			want:          "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789012/resourceGroups/testResourceGroup/providers/Microsoft.Provider/resourceTypes/resourceName/managedNamespaces/dev/providers/Microsoft.Authorization/checkaccess?api-version=2018-09-01-preview",
+			want:          "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789abc/resourceGroups/testResourceGroup/providers/Microsoft.Provider/resourceTypes/resourceName/managedNamespaces/dev/providers/Microsoft.Authorization/checkaccess?api-version=2018-09-01-preview",
+		},
+		{
+			name:          "valid with fleet resource ID without namespace",
+			baseURL:       mustCreateURL("https://management.azure.com"),
+			resourceID:    "/subscriptions/12345678-1234-1234-1234-123456789abc/resourceGroups/testResourceGroup/providers/Microsoft.ContainerService/fleets/my-fleet",
+			hasNamespace:  false,
+			namespacePath: "",
+			want:          "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789abc/resourceGroups/testResourceGroup/providers/Microsoft.ContainerService/fleets/my-fleet/providers/Microsoft.Authorization/checkaccess?api-version=2018-09-01-preview",
+		},
+		{
+			name:          "valid with fleet resource ID with namespace",
+			baseURL:       mustCreateURL("https://management.azure.com"),
+			resourceID:    "/subscriptions/12345678-1234-1234-1234-123456789abc/resourceGroups/testResourceGroup/providers/Microsoft.ContainerService/fleets/my-fleet",
+			hasNamespace:  true,
+			namespacePath: namespaces + "/dev",
```

Comment:
looks like an almost identical test to the one below, but this one isn't actually used in the code.  Remove this test case?